sources/platform/actors/development/container_web_server.md

```diff
@@ -0,0 +1,42 @@
+---
```

Comment:
This was originally part of the "Running your actors" which does not make any sense so I moved this to "Development" section and will improve update it in the next round.